6. Developer Guides/09. Realtime-API/1. Method Calls/12. Send Message/README.md

```diff
@@ -0,0 +1,96 @@
+# Send Message
+
+There are a few ways to comunicate when using Rocket.Chat, as desbribed bellow.
+
+## Sending a text message
+
+The most basic (and more common) way to comunicate is throught plain text. You're going to use the `sendMessage` method and send a single parameter: a `message`.
+
+The `message` is described as:
+
+- `_id`: The message id
+- `rid`: The room id for which this message is been send
+- `msg`: The message body (the text message itself)
+
+```json
+{
+    "msg": "method",
+    "method": "sendMessage",
+    "id": "42",
+    "params": [
+        {
+            "_id": "message-id",
+            "rid": "room-id",
+            "msg": "Hello World!"
+        }
+    ]
+}
+```
+
+## Sending a file
+
+(How I known I have to use S3? From the settings?)
```

Comment:
@laggedHero I think I understand what you're asking, you're wondering how you can determine if the server is using Amazon S3 or something else? If so, then the setting you're looking for is `FileUpload_Storage_Type` and the current valid settings are:

- GridFS
- AmazonS3
- FileSystem